### DISCUSSION TITLE
Question about Microsoft Fabric Assets License

### DISCUSSION THREAD

**Discussion by AkiraVoid** [microsoft/fluentui#23452] [Q&A] [answered]
Can I use these assets published by Microsoft Fabric Assets License in a business product without making profits on them directly (only for displaying)?

I mean, display them to make information clearer to users can also be regard as I have made profits on them.

**Comment by micahgodbolt** [comment, marked as answer]
Be sure to take a look at our License fully https://static2.sharepointonline.com/files/fabric/assets/microsoft_fabric_assets_license_agreement_nov_2019.pdf

But it is my understanding that you are welcome to use those assets however you like as long as you

>...always comply with the specifications, standards and directions relating to the use of Microsoft’s 
trademarks and brands set out in https://www.microsoft.com/en-us/legal/intellectualproperty/trademarks
